src/database/migrations/20230214055456-create-otps.js

```diff
@@ -0,0 +1,49 @@
+/** @type {import('sequelize-cli').Migration} */
```

Comment:
ah I wasn't aware of this requirement.

@alexanderleegs was this requirement present before as well? should we modify to reset after the `last_sent + REQ_WAIT_TIME`?
If it is within this buffer time, we should reject reset requests. Do we send a BadReqError for that?

Reply:
Ah right this was something I missed as well - right now there's a restriction on the frontend for a re-request to only be sent after 1 minute, but calling the endpoint directly to request for a new otp repeatedly is still possible.

I'm wondering if we should leave this as is though, since the new rate limiter on this endpoint stops the user from requesting too often, and we also might not want to block a user who accidentally navigates away from the page after requesting for the otp from requesting a new one. What are your thoughts @seaerchin?

Reply:
i think the intent of gating the otp requests is to prevent potential abuse (try -> fail -> request new otp -> start again); i think as long as we prevent this specific form of abuse (w/ a frontend gate so lawful users still adhere to a reasonable limit), it's ok if we remove this particular one (as the rate limiter still gates it)

Reply:
Okay, excluding this since we have rate limiter in place to prevent brute force or abuse

@seaerchin @alexanderleegs what's the rate limiter config though? Where can i find it?

Reply:
#610

Reply:
Thanks @seaerchin